frontend/app/src/components/inputs/AmountInput.vue

```diff
@@ -46,20 +46,15 @@ export default defineComponent({
   setup(props, { emit }) {
     const { integer, value } = toRefs(props);
 
-    const textInput = ref(null);
+    const { thousandSeparator, decimalSeparator } = setupSettings();
 
-    const currentValue = ref(get(value));
+    const formattedValue = get(value).replace('.', get(decimalSeparator));
```

Comment:
So the only question I have here is, do we know if `toFixed` is affected by the user's locale? because if it is, then this will not work in all cases.

Reply:
ok it seems to be hardcoded in the library so we should be safe